docs/install.md

```diff
@@ -24,11 +34,19 @@ sleep 30 # Wait for the analyzer model to load
 docker run --rm --name presidio-api --network mynetwork -d -p 8080:8080 -e WEB_PORT=8080 -e ANALYZER_SVC_ADDRESS=presidio-analyzer:3000 -e ANONYMIZER_SVC_ADDRESS=presidio-anonymizer:3001 -e RECOGNIZERS_STORE_SVC_ADDRESS=presidio-recognizers-store:3004 ${DOCKER_REGISTRY}/presidio-api:${PRESIDIO_LABEL}
 ```
 
-**NOTE: Building the deps images currently takes some time** (~70 minutes, depending on the build machine). We are working on improving the build time through improving the build and providing pre-built dependencies.
+Once the build is complete you can verify the local deployment by running `docker ps`. You should see 4 Presidio containers and one Redis container running with the following names:
+
+```sh
+presidio-api
+presidio-recognizers-store
+presidio-anonymizer
+presidio-analyzer
+redis
```

Comment:
Related to #239 
@eladiw Do we have a functional test which checks the recognizer store in a Docker installation?